docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -49,11 +48,11 @@ binaryData:
 EOF
 ----
 
-. Patch `OpenStackControlPlane` to deploy the {object_storage}:
+. Patch the `OpenStackControlPlane` custom resource to deploy the {object_storage}:
```

Comment:
@cschwede Which file is being updated in this step? Is it the openstack_control_plane.yaml file, which defines the OpenStackControlPlane CR?

Reply:
Yes, exactly that file. One can either patch an already existing CR (like in this example case) or patch the file before creating the CR.